tags/file_manager/file_manager.py

```diff
@@ -69,6 +73,11 @@
 ctx.lists["self.file_manager_files"] = []
 
 
+@ctx_file_manager.capture("user.address", rule="<user.system_diretory>")
```

Comment:
Could we go directly to the system directory list instead of the capture? This appears to be done with the web browser context.

Reply:
Fixed. Mismerge from my local stuff where user.system_diretory actually exists